libs/common/src/types/mongo-db-models/tfm-facilities.ts

```diff
@@ -120,12 +121,20 @@ export interface TfmFacilityAmendment extends BaseAmendment {
   };
 }
 
+export type AmendmentsEligibilityCriterionWithAnswer = AmendmentsEligibilityCriterion & {
```

Comment:
```suggestion
export type AmendmentsEligibilityCriterionWithAnswer = AmendmentsEligibilityCriterion && {
```

Reply:
I don't think that's valid syntax here for types - what's the reason for the `&&` suggetsion?

Reply:
Apologies, I was trying to type below would it be possible to add `answer` optional property to the type rather than using type intersection here?

Reply:
It's important in certain places we know the answer field is specified and is null or boolean. In particular there are places where it is _not_ optional and it's good to keep track of this.

So keen to keep it as is